xml/troubleshooting.xml

```diff
@@ -1079,64 +1079,67 @@
-    work, the screen geometry becoming distorted, or even the screen coming up
-    as a bare gray field. The important distinction is that if another user
-    logs in, the machine works normally. It is then likely that the problem can
-    be fixed relatively quickly by simply moving the user's &gnome;
-    configuration directory to a new location, which causes &gnome; to
-    initialize a new one. Although the user is forced to reconfigure &gnome;,
-    no data is lost.
+     
+     If you are experiencing issues with the &gnome; desktop (for
+     example, incorrect screen geometry, blank desktop, etc.), there
+     are several ways to troubleshoot the misbehaving graphical
+     desktop environment. The recommended procedure described below
+     offers the safest option to fix broken &gnome;.
    </para>
-   <procedure xml:id="proc-trouble-login-dk">
-    <step>
-     <para>
-      Switch to a text console by pressing <keycombo>
-      <keycap function="control"/> <keycap function="alt"/> <keycap>F1</keycap>
-      </keycombo>.
-     </para>
-    </step>
-    <step>
-     <para>
-      Log in with your user name.
-     </para>
-    </step>
-    <step>
-     <para>
-      Move the user's &gnome; configuration directories to a temporary
-      location:
-     </para>
-<screen>&prompt.user;mv .gconf  .gconf-ORIG-RECOVER
-&prompt.user;mv .gnome2 .gnome2-ORIG-RECOVER</screen>
-    </step>
-    <step>
-     <para>
-      Log out.
-     </para>
-    </step>
-    <step>
-     <para>
-      Log in again, but do not run any applications.
-     </para>
-    </step>
-    <step>
-     <para>
-      Recover your individual application configuration data (including the
-      Evolution e-mail client data) by copying the
-      <filename>~/.gconf-ORIG-RECOVER/apps/</filename> directory back into the
-      new <filename>~/.gconf </filename> directory as follows:
-     </para>
-<screen>&prompt.user;cp -a .gconf-ORIG-RECOVER/apps .gconf/</screen>
-     <para>
-      If this causes the login problems, attempt to recover only the critical
-      application data and reconfigure the remainder of the applications.
-     </para>
-    </step>
+   <procedure>
+     <title>Troubleshooting &gnome;</title>
+     <step>
+       <para>
+	 Launch &yast; and switch to <guimenu>Security and Users</guimenu>.
+       </para>
+     </step>
+     <step>
+       <para>
+	 Open the <guimenu>User and Group Management</guimenu> dialog and press <guimenu>Add</guimenu>.
+       </para>
+     </step>
+     <step>
+       <para>
+	 Fill out the required fields and press <guimenu>OK</guimenu> to create a new user.
+       </para>
+     </step>
+     <step>
+       <para>
+	 Log out and log in as the new user. This gives with a fresh
+	 installation of &gnome;.
+       </para>
+     </step>
+     <step>
+       <para>
+	 Copy individual &gnome; &gnome;-specific configuration files from the
+	 <filename>.local/.config/</filename> directory of the old
```

Comment:
As I understood what it implies here, perhaps you meant .local and .config directories under the user's home? So they are two different directories. The format in line 1117 looks like they are one single directory.

Besides, I remember Stefan mentioned similar thing in an earlier PR, we'd better to guide the user to the corresponding path (eg. cd $HOME) before using the relative path to explain things. Please ignore this if
it has been mentioned in the context around the changing segments.

Reply:
My bad, these are indeed two different directories.

> Besides, I remember Stefan mentioned similar thing in an earlier PR, we'd better to guide the user to the corresponding path (eg. cd $HOME)

True, but the reader needs to copy files from the old user account to the new one. So using $HOME is not practical here. But I'll see if I can do something about it.